qa/workunits/cephadm/test_iscsi_pids_limit.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# checks if the containers default pids-limit (4096) is removed and Iscsi
+# containers continue to run
+# exists 1 if fails
+
+SCSI_CONT_IDS=$(podman ps -qa --filter='name=iscsi')
+
+for i in $SCSI_CONT_IDS
+do
+  podman exec ${i} /bin/sh -c 'for j in {0..5000}; do sleep 30& done'
+done
+
+for i in $SCSI_CONT_IDS
+do
+  SLEEP_COUNT=$(podman exec ${i} /bin/sh -c 'ps -ef | grep sleep | wc -l')
+  test $SLEEP_COUNT -gt 5000
+done
```

Comment:
```suggestion
for i in $SCSI_CONT_IDS
do
  podman exec ${i} /bin/sh -c 'for j in {0..5000}; do sleep 30& done'
  SLEEP_COUNT=$(podman exec ${i} /bin/sh -c 'ps -ef | grep sleep | wc -l')
  test $SLEEP_COUNT -gt 5000 || (echo "The container ${i} didn't pass the iscsi pids limit test" && exit 1)
done
```

Reply:
As discussed, I will leave it as is. Thx